src/Uno.UI.RuntimeTests/Tests/Windows_UI/Given_Color.cs

```diff
@@ -63,5 +63,25 @@ public static IEnumerable<object[]> GetCompare()
 			yield return new object[] { ColorHelper.FromArgb(0xFF, 0xFF, 0x7F, 0xFF), ColorHelper.FromArgb(0xFF, 0xFF, 0x7F, 0xFF), true };
 			yield return new object[] { ColorHelper.FromArgb(0xFF, 0xFF, 0xFF, 0x7F), ColorHelper.FromArgb(0xFF, 0xFF, 0xFF, 0x7F), true };
 		}
+#if __MACOS__
+		[TestMethod]
+		public async Task When_User_Change_macOS_System_Colors()
+		{
+			var _uiSettings = new Windows.UI.ViewManagement.UISettings();
+
+		  	Color SUT_1 = _uiSettings.GetColorValue(Windows.UI.ViewManagement.UIColorType.Accent);
+			Color SUT_2 = _uiSettings.GetColorValue(Windows.UI.ViewManagement.UIColorType.Background);
+
+			var accent = new Windows.UI.Xaml.Media.SolidColorBrush(AppKit.NSColor.ControlAccentColor).Color;
```

Comment:
This is the right name:
```suggestion
			var accent = new Windows.UI.Xaml.Media.SolidColorBrush(AppKit.NSColor.ControlAccent).Color;
```